app/views/exchange_rates/_document_detail.html.erb

```diff
@@ -12,21 +12,18 @@
         <%= Date::MONTHNAMES[period.month] %> <%= period.year %> <%= type %> exchange rates
       </h3>
 
-      <p>
-        <strong>
-          <% if period.has_exchange_rates %>
-            <%= link_to "View online", exchange_rate_collection_path("#{period.year}-#{period.month}", type: type), class: 'govuk-link' %>
-
-          <% end %>
-        </strong>
+      <p class="document-download">
+        <% if period.has_exchange_rates %>
+          <%= link_to "<strong>View online</strong>".html_safe, exchange_rate_collection_path("#{period.year}-#{period.month}", type: type), title: "View #{period.month} #{period.year} monthly exchange rates online", class: 'govuk-link' %>
```

Comment:
Use of `.html_safe` is generally reserved for 'there's not another option' - it can cause problems if not used correctly and whilst this use is safe - usage of it in a code base tends to then lead to further usage.

Here it is possible to use regular rails helpers, this could be rewritten using either the `tag.strong('View online')` in place of the string or more neatly by using a code block onto the end of link_to

```erb
<% link_to exchange_rate_collection_path(), title: "View ..." do %>
  <strong>View online</strong>
<% end %>
```

The addition of `govuk-link` does no harm but isn't actually needed because we automatically add that to all links anyway (see https://github.com/trade-tariff/trade-tariff-frontend/blob/main/app/webpacker/packs/application.scss#L1)